apps/dh/api-dh/source/DataHub.WebApi/Registration/GraphQLRegistrationExtensions.cs

```diff
@@ -27,6 +27,7 @@ public static IRequestExecutorBuilder AddGraphQLServices(this IServiceCollection
     {
         return services
             .AddGraphQLServer(disableDefaultSecurity: true)
+            .InitializeOnStartup()
```

Comment:
Is this to speed up the first query or? 🙂

Reply:
Yes